src/server/game/Entities/Conversation/Conversation.cpp

```diff
@@ -130,7 +131,22 @@ bool Conversation::Create(ObjectGuid::LowType lowGuid, uint32 conversationEntry,
             actorField.Type = ConversationDynamicFieldActor::ActorType::CreatureActor;
             SetDynamicStructuredValue(CONVERSATION_DYNAMIC_FIELD_ACTORS, actorsIndex++, &actorField);
         }
-        else
+        else if (!conversationTemplate->Actors.empty())
```

Comment:
this won't work if you have a mix of both actor types, you need to start from 0 when adding actors with guid

Reply:
you are right. Gonna fix that